2_eselect/eselect.yaml

```diff
@@ -1,6 +1,7 @@
 moduleIdentifier:
   moduleName: eselect
   moduleVersion: 0.0.1
+  namespace: test/abc

```

Comment:
`namespace` is optional. Let's remove it from this yaml to test the case.

Reply:
OK. Please also help update the definition here: https://msdata.visualstudio.com/AzureML/_git/ModuleDocs?path=%2Fdocs%2Fdefinition-of-module-spec.md&_a=preview

Reply:
Updated